### PR TITLE
Update morpho v3

### PR DESCRIPTION
**Pull request by shriyatyagii**
image

### CODE REVIEW

File: contracts/mainnet/connectors/morpho-aave-v3/helpers.sol [hunk 1]
```diff
@@ -10,7 +10,7 @@ abstract contract Helpers is Stores, Basic {
 	IMorphoCore public constant MORPHO_AAVE_V3 =
 		IMorphoCore(0x33333aea097c193e66081E930c33020272b33333);
 	
-	uint256 public max_iteration = 4;
+	uint256 public constant max_iteration = 4;
```
Comment (thrilok209) [inline contracts/mainnet/connectors/morpho-aave-v3/helpers.sol]:
```suggestion
	uint256 public constant MAX_ITERATIONS = 4;
```

File: contracts/mainnet/connectors/morpho-aave-v3/main.sol [hunk 2]
```diff
@@ -737,13 +655,13 @@ abstract contract MorphoAaveV3 is Helpers, Events {
 		external
 		returns (string memory _eventName, bytes memory _eventParam)
 	{
-		uint256 _oldIterations = max_iteration;
-		max_iteration = _iterations;
+		uint256 _oldIterations = MAX_ITERATIONS;
```
Comment (thrilok209) [inline contracts/mainnet/connectors/morpho-aave-v3/main.sol]: @shriyatyagii Remove this function.

File: contracts/mainnet/connectors/morpho-aave-v3/helpers.sol (hunk 1 above)
Comment (shriyatyagii) [reply contracts/mainnet/connectors/morpho-aave-v3/helpers.sol]: Updated

File: contracts/mainnet/connectors/morpho-aave-v3/main.sol (hunk 2 above)
Comment (shriyatyagii) [reply contracts/mainnet/connectors/morpho-aave-v3/main.sol]: Removed

**Comment by shriyatyagii** [comment]
Update connector details ->

- `MORPHO-AAVE-V3-A`
- New address: [0x5bb2daA4A140Ae5Ff4aeE77F5Bb949e4f9e69DAc](https://etherscan.io/address/0x5bb2daA4A140Ae5Ff4aeE77F5Bb949e4f9e69DAc#code)
- Txn: [0xbca998bfd059c8507a1e9ed71cedbb5ae2f5641a28eb52375f9b3c7d6ed3bc9e](https://etherscan.io/tx/0xbca998bfd059c8507a1e9ed71cedbb5ae2f5641a28eb52375f9b3c7d6ed3bc9e)

**Review by thrilok209 — APPROVED** [review]

**Pull request merged**